### ISSUE TITLE
compatibility with 2019

### ISSUE THREAD

**Issue by SingingBush** [intellij-dlanguage/intellij-dlanguage#482]
2019 will be some breaking changes as well as deprecation's marked for removal in 2020. Changes to be done in [2019 feature branch](https://github.com/intellij-dlanguage/intellij-dlanguage/tree/feature/2019_support ) and get merged after next release. (Still need to sort #481 and publish).

**Comment by SingingBush** [comment]
some of these changes were able to go into v1.18 so we have initial 2019.1 support, although there's a ton of warnings. The 2019 branch should get used to clear up all those deprecation warnings

**Comment by SingingBush** [comment]
Also Kotlin will be version 1.3: https://www.jetbrains.org/intellij/sdk/docs/reference_guide/api_changes/api_changes_list_2019.html

**Comment by SingingBush** [comment]
The feature branch for 2019 support has been rebased and merged into master. All work done now should be targeting 2019.1 as minimum

**Comment by HydrogenC** [comment]
Version for 2019.1 released on plugins.jetBrains.com?

**Comment by SingingBush** [comment]
I've not had time to work on the project for a while but from what I remember there's still more to do before the plugin can work properly on 2019.*

I don't expect to get time to look at it this month so any help with testing/fixing problems would be good.

**Comment by aesy** [comment]
I'd like to help. Any pointers as to what needs to be tested/fixed?

**Comment by SingingBush** [comment]
I ran through the basics today, just creating a project and building/running with dmd. Also checked that DCD integration looked ok. Seems ok but I was fairly sure I'd seen issues before. I've tagged a release of 1.19 anyway.

As stated in previous comment, changes are going to be needed to prevent the plugin from breaking at some point soon.

@aesy much of the functionality cannot be tested with automated tests so it can be very helpful having people running the code through a debugger and going through the general use cases of creating/opening projects and using the various D Tools
